recipes/cupy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cupy" %}
+{% set version = "6.4.0" %}
+{% set sha256 = "ca70ca0823323f8c91d52eb304009eaa72feb44118864c1f00d8fa288352dc88" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not (linux64 and cuda_compiler_version == "9.2")]
+  ignore_run_exports:
+    - fastrlock
+  missing_dso_whitelist:
+    - "*/libcuda.*"
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("cuda") }}
+
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython >=0.24.0
+    - cudnn
+    - fastrlock >=0.3
+    - nccl
+
+  run:
+    - python
+    - setuptools
+    - {{ pin_compatible("cudnn") }}
+    - fastrlock >=0.3
+    - numpy >=1.9.0
+    - six >=1.9.0
+
+# Requires a GPU to import (loads libcuda)
```

Comment:
Do you need a GPU or just `libcuda`? AFAIK, the docker image has the drivers (which includes libcuda) installed.

Reply:
We actually need a GPU. This is for running the full test suite, which executes on a GPU.

Also `libcuda` here means the driver library.

Reply:
Okay. So you can't even do `import cupy` without a GPU?

Reply:
Unfortunately not. Please see PR ( https://github.com/cupy/cupy/pull/1781 ), which is trying to loosen this constraint.

There is some code in `run_test.py` that uses `pkgutil` to find the `cupy` module without actually importing it.

Reply:
FWIW I've moved this also into the `run_test.py` and made the tests run conditionally if a GPU is present. Should make it easier for people building with hardware.